<launch>    
  <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>


  <!--- Run AMCL -->
  <include file="$(find amcl)/examples/amcl_diff.launch" />
<!-- Define your move_base node -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find robot_navigation)/cost_map/costmap_common_params_$(arg model).yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find robot_navigation)/cost_map/costmap_common_params_$(arg model).yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find robot_navigation)/cost_map/local_costmap_params.yaml"  command="load" />
    <rosparam file="$(find robot_navigation)/cost_map/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find robot_navigation)/cost_map/base_local_planner_params.yaml" command="load" />
    <remap from="map" to="/map" />
  </node>
</launch>
